plugins/plugin-site/src/components/PluginPageLayout.jsx

```diff
@@ -104,7 +104,7 @@ function PluginPageLayout({plugin, children}) {
                         <h5>Maintainers</h5>
                         <PluginDevelopers developers={plugin.developers} />
                     </div>
-                    {shouldShowWikiUrl(plugin.wiki) &&
+                    {shouldShowWikiUrl(plugin.wiki?) &&
```

Comment:
```suggestion
                    {shouldShowWikiUrl(plugin.wiki || {}) &&
```

or we fix it upstream in the source plugin